appservice/src/createAppService/createAppService.ts

```diff
@@ -35,6 +36,13 @@ export async function createAppService(
         environment: node.environment
     };
 
+    if (actionContext.properties.resourceGroup) {
```

Comment:
Same comment as other PR:

`actionContext.properties` is used for telemetry and we don't want to collect this in telemetry

We'll have to find a different way to pass the resource group along